avm/res/api-management/service/api/main.bicep

```diff
@@ -207,7 +207,6 @@ output resourceGroupName string = resourceGroup().name
 import * as operationTypes from 'operation/main.bicep'
 
 @export()
-@sys.description('The type of an operation.')
```

Comment:
Why the removal? These descriptions are displayed when importing the type from another file 😄